packages/ui-pill/src/Pill/index.tsx

```diff
@@ -95,6 +95,7 @@ class Pill extends Component<PillProps, PillState> {
   }
 
   renderPill(
+    needsRef: boolean,
```

Comment:
Same as in NavigationItem: I don't think "needsRef" is self-explanatory enough, I would use `this.state.truncated` here too (and/or some comment/explanation above it)

Reply:
done